packages/next/src/executors/build/lib/create-package-json.ts

```diff
@@ -39,10 +36,7 @@ export async function createPackageJson(
   writeJsonFile(`${options.outputPath}/package.json`, packageJson);
 
   // generate lock file
-  const prunedLockFile = pruneLockFileFromPackageJson(
-    packageJson,
-    !options.includeDevDependenciesInPackageJson
-  );
+  const prunedLockFile = createLockFile(packageJson);
```

Comment:
We don't need the option here anymore because it is just based off the `package.json` that's sent in right?